.github/workflows/tests.yml

```diff
@@ -22,9 +22,38 @@ jobs:
         pip3 install -r requirements.txt
```

Comment:
Given these tests are bound to docker workflow, rename from `.github/workflows/tests.yml` to `.github/workflows/tests-docker.yml`